sessions/js-unit-testing/math-functions_solution/index.test.js

```diff
@@ -0,0 +1,67 @@
+import { add, subtract, multiply, divide } from "./index.js";
+
+test("returns 5 if called with add(2, 3)", () => {
+  const result = add(2, 3);
+
+  expect(result).toBe(5);
+});
+
+test("returns a negative value if the greater argument is negative", () => {
+  const result = add(2, -3);
+
+  expect(result).toBeLessThan(0);
+});
+
+test("returns a value close to 0.3 if called with add(0.1, 0.2)", () => {
+  const result = add(0.1, 0.2);
+
+  expect(result).toBeCloseTo(0.3);
+});
+
+test("returns 10 if called with subtract(15, 5)", () => {
+    const result = subtract(15, 5);
+
+    expect(result).toBe(10);
+});
+
+test("returns a negative value if the second argument is greater than the first one", () => {
+    const result = subtract(5, 15);
+
+    expect(result).toBeLessThan(0);
+});
+
+test("returns 8 if called with multiply(2, 4)", () => {
+    const result = multiply(2, 4);
+
+    expect(result).toBe(8);
+});
+
+test("returns a negative value if only the first argument is negative", () => {
+    const result = multiply(-2, 4);
+
+    expect(result).toBeLessThan(0);
+});
+
+test("returns a negative value if only the second argument is negative", () => {
+    const result = multiply(2, -4);
+
+    expect(result).toBeLessThan(0);
+});
+
+test("returns a positive value if called with two negative arguments", () => {
+    const result = multiply(-2, -4);
+
+    expect(result).toBeGreaterThan(0);
+});
```

Comment:
I was thinking that we should show an example where multiple expect statements are in one test. Since these all have the same structure students might come to the conclusion that you have to obey this "one test one expect" structure.

Reply:
Hi Felix,

I think it is a good point, but it would mean adjusting the challenge.

Currently we give them the criteria to test for and we ask them to write a test for each of them.

In general, I like having separate well documented tests for complex tests and with simple tests as in this challenge, it makes sense to combine them into one test. We could even consider adjusting the demo a little bit, to have one test with several expects.

I would propose this: 
I would merge the current solutions, since they match what we currently show students.
I would also create a small ticket to adjust the Unit Testing Session, so that we show a test with several expects in the demo and also with an adjustment to the math challenge. 
What do you think?

Reply:
that sounds perfect, lets do it like that.